plugins/upstash/test-fixtures/upstash.json

```diff
@@ -0,0 +1,4 @@
+{
+    "apiKey":"d68850db-69f7-2b87x3f5u7sxinsexample",
+    "email" : "fakeemail12@gmail.com"
+}
```

Comment:
nit: let's add a new line to this file

Reply:
This doesn't seem to be an issue anymore.